simphony/cuds/abc_dataset.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+from abc import ABCMeta, abstractmethod
+
+
+class ABCDataset(object):
```

Comment:
Ok then I probably used the wrong name. I just picked dataset but I am not sure about it.
What is the generic name for things like particles, mesh, and lattice?